### PR TITLE
Passing errors refactoring

### PR DESCRIPTION
**Pull request by bmallred**

### CODE REVIEW

File: src/components/Form/Number/Number.jsx [hunk 1]
```diff
@@ -144,18 +106,33 @@ export default class Number extends ValidationElement {
...
   pattern: '^(\\s*|\\d+)$',
-  focus: false,
-  error: false,
-  valid: false,
-  errorCode: null
+  errorCode: null,
+  onError: (value, arr) => { return arr }
 }
+
+Number.errors = [
```
Comment (floresj) [inline src/components/Form/Number/Number.jsx]: Every time I see this it makes me smile.

**Comment by bmallred** [comment]
@floresj just FYI... I'm starting to merge in the changes from `master` and there are about 27 conflicts which need to be resolved. If we can postpone any significant changes to `master` for a bit that would be great :smile:

**Comment by floresj** [comment]
### Information about you
- Entire section marked as done when the following sections are completed
  -  Full name, Your contact information, other names used
selection_141
  -  Full name and other names used
selection_144


- Your Identifying information icon lays on top of text when completed
selection_143

File: src/components/Section/History/History.jsx [hunk 2]
```diff
@@ -32,149 +31,115 @@ const byline = (item, index, initial, translation, validator) => {
...
-  /**
-   * Update storage values for a subsection
-   */
-  onUpdate (field, values) {
-    this.props.dispatch(updateApplication('History', field, values))
-  }
+    this.overrideInitial = this.overrideInitial.bind(this)
+  }
+
+  // /**
```
Comment (floresj) [inline src/components/Section/History/History.jsx]: Cool if I remove this comment block?

Comment (bmallred) [inline src/components/Section/History/History.jsx]: Absolutely... kept it around just b/c the logic was so different than the rest of them. Think that is all sync'd now.

**Pull request merged**